Server/BackEndMonitoringServer/include/CProcess.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <windows.h>
```

Comment:
Move to cpp

Reply:
moved everything beside custom file (EMemoryCountType)